01-html/04-nytimes.html

```diff
@@ -1,3 +1,1120 @@
+          <li><a href="http://www.nytimes.com/pages/opinion/index.html#editorials">Editorials</a></li>
+          <li><a href="http://www.nytimes.com/pages/opinion/index.html#contributing">Contributing Writers</a></li>
+          <li><a href="http://www.nytimes.com/pages/opinion/index.html#op-ed">Op-Ed Contributors</a></li>
+          <li><a href="http://www.nytimes.com/pages/opinion/index.html#opinionator">Opinionator</a></li>
+          <li><a href="http://www.nytimes.com/pages/opinion/index.html#letters">Letters</a></li>
+          <li><a href="http://www.nytimes.com/pages/opinion/index.html#sundayreview">Sunday Review</a></li>
+          <li><a href="http://www.nytimes.com/pages/opinion/index.html#takingNote">Taking Note</a></li>
+          <li><a href="http://www.nytimes.com/roomfordebate">Room for Debate</a></li>
+          <li><a href="http://topics.nytimes.com/top/opinion/thepubliceditor/index.html">Public Editor</a></li>
+        </ul>
+        <li><a href="http://www.nytimes.com/pages/technology/index.html">Tech</a></li>
+        <ul>
+          <li><a href="http://bits.blogs.nytimes.com/">Bits</a></li>
+          <li><a href="http://www.nytimes.com/pages/technology/personaltech/index.html">Personal Tech</a></li>
+        </ul>
+        <li><a href="http://www.nytimes.com/section/science">Science</a></li>
+        <ul>
+          <li><a href="http://www.nytimes.com/pages/science/earth/index.html">Environment</a></li>
+          <li><a href="http://www.nytimes.com/pages/science/space/index.html">Space & Cosmos</a></li>
+        </ul>
+        <li><a href="http://www.nytimes.com/pages/health/index.html">Health</a></li>
+        <ul>
+          <li><a href="http://well.blogs.nytimes.com/">Well</a></li>
+          <li><a href="http://www.nytimes.com/pages/health/policy/index.html">Money & Policy</a></li>
+          <li><a href="http://www.nytimes.com/health/guides/index.html">Health Guide</a></li>
+        </ul>
+        <li><a href="http://www.nytimes.com/pages/sports/index.html">Sports</a></li>
+        <ul>
+          <li><a href="http://www.nytimes.com/pages/sports/baseball/index.html">Baseball</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/ncaabasketball/index.html">Basketball: College</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/basketball/index.html">Basketball: N.B.A.</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/ncaafootball/index.html">Football: College</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/football/index.html">Football: N.F.L.</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/golf/index.html">Golf</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/hockey/index.html">Hockey</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/soccer/index.html">Soccer</a></li>
+          <li><a href="http://www.nytimes.com/pages/sports/tennis/index.html">Tennis</a></li>
+        </ul>
```

Comment:
You may notice a little border here. NYTimes groups these categories into sections, so it would be appropriate to divide them into `section`s in the nave as well.